proto/shentu/bounty/v1/bounty.proto

```diff
@@ -42,13 +42,15 @@ message Finding {
   uint64 finding_id = 1 [(gogoproto.jsontag) = "id", (gogoproto.moretags) = "yaml:\"id\""];
   string title = 2 [(gogoproto.moretags) = "yaml:\"title\""];
   google.protobuf.Any encrypted_desc = 3 [(cosmos_proto.accepts_interface) = "EncryptedDesc",(gogoproto.moretags) = "yaml:\"encrypted_desc\""];
-  uint64 program_id = 4 [(gogoproto.moretags) = "yaml:\"program_id\""];
-  SeverityLevel severity_level = 5 [(gogoproto.moretags) = "yaml:\"severity_level\""];;
-  google.protobuf.Any encrypted_poc = 6 [(cosmos_proto.accepts_interface) = "EncryptedPoc",(gogoproto.moretags) = "yaml:\"encrypted_poc\""];
-  string submitter_address  = 7 [(gogoproto.moretags) = "yaml:\"submitter_address\""];
-  FindingStatus finding_status = 8 [(gogoproto.moretags) = "yaml:\"finding_status\""];
-
-  google.protobuf.Any encrypted_comment = 9 [(cosmos_proto.accepts_interface) = "EncryptedComment",(gogoproto.moretags) = "yaml:\"encrypted_comment\""];
+  string  desc = 4 [(gogoproto.moretags) = "yaml:\"desc\""];
+  uint64 program_id = 5 [(gogoproto.moretags) = "yaml:\"program_id\""];
+  SeverityLevel severity_level = 6 [(gogoproto.moretags) = "yaml:\"severity_level\""];;
+  google.protobuf.Any encrypted_poc = 7 [(cosmos_proto.accepts_interface) = "EncryptedPoc",(gogoproto.moretags) = "yaml:\"encrypted_poc\""];
+  string poc = 8 [(gogoproto.moretags) = "yaml:\"poc\""];
+  string submitter_address  = 9 [(gogoproto.moretags) = "yaml:\"submitter_address\""];
+  FindingStatus finding_status = 10 [(gogoproto.moretags) = "yaml:\"finding_status\""];
+  google.protobuf.Any encrypted_comment = 11 [(cosmos_proto.accepts_interface) = "EncryptedComment",(gogoproto.moretags) = "yaml:\"encrypted_comment\""];
+  string comment = 12 [(cosmos_proto.accepts_interface) = "Comment",(gogoproto.moretags) = "yaml:\"comment\""];
```

Comment:
I think encrypted_poc & poc or encrypted_comment & comment pairs can be merged to accept an interface?

Reply:
yes, I will merge these.